### PR TITLE
Styles for ML Carbon console

### PR DESCRIPTION
**Pull request by manoramahp**

### CODE REVIEW

File: modules/features/product/feature.properties [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+      incurred by, or claims asserted against, such Contributor by reason\n\
+      of your accepting any such warranty or additional liability.\n\
+\n\
+   END OF TERMS AND CONDITIONS\n
+
+######################### copyright properties #################################
+copyrightURL=TODO
+
+copyright=\
+Copyright 2009 WSO2, Inc. (http://wso2.com)\n\
```
Comment (nirmal070125) [inline modules/features/product/feature.properties]: Should this be 2015?

File: modules/features/product/org.wso2.ml.styles.feature/pom.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    <parent>
+        <groupId>org.wso2.ml</groupId>
+        <artifactId>wso2ml-features</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.ml.styles.feature</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (nirmal070125) [inline modules/features/product/org.wso2.ml.styles.feature/pom.xml]: Version should be inherited from parent.

File: modules/styles/product/src/main/resources/web/product/about.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  ~ limitations under the License.
+  -->
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<html><head><title>WSO2 ML - About</title>
+
+<link href="../admin/css/documentation.css" rel="stylesheet" type="text/css" media="all"/>
+</head>
+<body>
+<h1>Version 4.0.0</h1>
```
Comment (nirmal070125) [inline modules/styles/product/src/main/resources/web/product/about.html]: Version 1.0.0 ?

Comment (manoramahp) [inline modules/styles/product/src/main/resources/web/product/about.html]: changed the version : https://github.com/manoramahp/product-ml/commit/bf9036e1245d6926db3daae5564ccc325f01e766

File: modules/features/product/org.wso2.ml.styles.feature/pom.xml (hunk 2 above)
Comment (manoramahp) [inline modules/features/product/org.wso2.ml.styles.feature/pom.xml]: removed the &lt;version&gt; : https://github.com/manoramahp/product-ml/commit/bf9036e1245d6926db3daae5564ccc325f01e766

File: modules/features/product/feature.properties (hunk 1 above)
Comment (manoramahp) [inline modules/features/product/feature.properties]: changed to 2015 : https://github.com/manoramahp/product-ml/commit/bf9036e1245d6926db3daae5564ccc325f01e766

**Comment by nirmal070125** [comment]
Thanks. Please merge.

**Pull request merged**